src/geoclient.js

```diff
@@ -2,6 +2,7 @@ import axios from 'axios';
 import axiosRetry from 'axios-retry';
 
 const { GEO_APP_KEY, GOOGLE_API_KEY } = process.env;
+console.log('XXX JMF', { GEO_APP_KEY, GOOGLE_API_KEY })
```

Comment:
This prints the following when I run `yarn start` locally, even when I set it on the CLI like this: `GEO_APP_KEY=CENSORED yarn start --release`

```
XXX JMF { GEO_APP_KEY: undefined, GOOGLE_API_KEY: undefined }
```

Reply:
fixed in https://github.com/josephfrazier/reported-web/pull/427/commits/2837fe8baca5e32a419d6c0c00e41c2d7aa74981